packages/mobile/src/store/store.ts

```diff
@@ -92,7 +93,8 @@ if (__DEV__) {
 export const store = createStore(
   createRootReducer(),
   applyMiddleware(...middlewares)
-)
+) as unknown as Store<AppState> // need to explicitly type the store for offline-mode store reference
```

Comment:
Oh, nice. This will fix the selector types on `store.getState`?

Could call that out more generally too since we may use it outside of offline mode in the future.

I was going to tackle this problem in general when exploring the new react-query selectors.

Reply:
yeah it does, def a bit hacky doing it like this, so maybe we can think of a more standard way in the future